modules/widgets/src/stylesheet.css

```
@@ -171,3 +181,8 @@
   );
 }
 
+.deck-widget-spinner-icon {
+  --deck-widget-icon-loading: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="black" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round" ><path d="M21 12a9 9 0 1 1-6.219-8.56" /></svg>');
+  mask-image: var(--deck-widget-icon-loading);
+  -webkit-mask-image: var(--deck-widget-icon-loading);
+}
```

```suggestion
.deck-widget-spinner-icon {
  mask-image: var(
    --loading-icon, 
    url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="black" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round" ><path d="M21 12a9 9 0 1 1-6.219-8.56" /></svg>')
  );
  -webkit-mask-image: var(
    --loading-icon,
    url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="black" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round" ><path d="M21 12a9 9 0 1 1-6.219-8.56" /></svg>')
  );
}
```

We're punting on #9481 so let's use the existing pattern